packages/t/tonic.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "tonic",
+  "description": "A composable component inspired by React.",
+  "keywords": [
+    "tonic",
+    "web-components",
+    "web",
+    "components",
+    "html"
+  ],
+  "license": "MIT",
+  "homepage": "https://tonic.technology/",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/optoolco/tonic.git"
+  },
+  "authors": [
+    {
+      "name": "Raynos",
+      "email": "raynos2@gmail.com",
+      "url": "https://github.com/optoolco/tonic/issues"
+    }
+  ],
+  "autoupdate": {
+    "source": "npm",
+    "target": "@optoolco/tonic",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "index.js"
+        ]
```

```suggestion
        "basePath": "dist",
        "files": [
          "*.js"
        ]
```

It looks like you provide a pre-bundled distributable version. We should use this instead as a Node version won't work on the web.

---

The pre-bundled variant is minified by default.

The Node version does happen to work on the web because the exports is a conditional

```
if (typeof module === 'object') module.exports = Tonic
```